pkg/console/subresource/deployment/deployment.go

```diff
@@ -270,6 +272,32 @@ func consoleContainer(cr *operatorv1.Console, volConfigList []volumeConfig) core
 	}
 }
 
+func setEnvironmentVariables(proxyConfig *configv1.Proxy) []corev1.EnvVar {
+	envVars := []corev1.EnvVar{}
+	if proxyConfig == nil {
+		return envVars
+	}
+	if len(proxyConfig.Status.HTTPSProxy) != 0 {
```

Comment:
You do a check on proxyConfig if it is nil, but what about proxyConfig.Status?  Is that guaranteed to be there?

Reply:
Shouldn't be needed since it's not a pointer